src/components/commanderOverview/CommanderOverview.tsx

```diff
@@ -37,7 +42,9 @@ export const CommanderOverview = React.memo(function MatchHistory() {
             (value: Commander) => allCommanders[value.id].matches.length >= COMMANDER_MINIMUM_GAMES_REQUIRED,
         );
     }
-
+    if (searchInput.length > 0 && allCommanders) {

```

Comment:
with this now being our second set of filter operations on the commandersArray, can we memoize the commandersArray? 

Feel free to break this out into a separate PR, but I think we can tackle this in just 1 PR.

Reply:
Won't fix for now. David found out this needs a bigger fix - `commander` needs to be also memoized. Currently it is returning a new array every time the page renders. So memoizing only commandersArray does nothing.